projects/1.12.2/assets/pneumaticcraft-repressurized/pneumaticcraft/patchouli_books/book/zh_cn/entries/programming/block_right_click.json

```
@@ -6,7 +6,7 @@
   "pages": [
     {
       "type": "text",
-      "text": "无人机将会对指定$(l:programming/area)区域/$内的每一个方块使用其物品栏内第一个物品$(thing)右击/$.这可以用来锄地，种下种子，施加骨粉，投掷药水，部署无人机，使用扳手旋转方块等...用它可以做到的可能性是无穷的。你可以通过放置一个指示某方块的$(l:programming/item_filter)物品过滤器/$来指定哪些方块应当被$(thing)右击/$。"
+      "text": "无人机将会对指定$(l:programming/area)区域/$内的每一个方块使用其物品栏内第一个物品$(thing)右击/$.这可以用来锄地，种下种子，施加骨粉，投掷药水，部署无人机，使用扳手旋转方块等……用它可以做到的可能性是无穷的。你可以通过放置一个指示某方块的$(l:programming/item_filter)物品过滤器/$来指定哪些方块应当被$(thing)右击/$。"
```

~自己修自己的错误~
（虽然不是省略号）
```suggestion
      "text": "无人机将会对指定$(l:programming/area)区域/$内的每一个方块使用其物品栏内第一个物品$(thing)右击/$。这可以用来锄地，种下种子，施加骨粉，投掷药水，部署无人机，使用扳手旋转方块等……用它可以做到的可能性是无穷的。你可以通过放置一个指示某方块的$(l:programming/item_filter)物品过滤器/$来指定哪些方块应当被$(thing)右击/$。"
```